WordPress/Classes/Utility/PageTree.swift

```diff
@@ -2,28 +2,35 @@ final class PageTree {
 
     // A node in a tree, which of course is also a tree itself.
     private class TreeNode {
-        let page: Page
+        struct PageData {
+            var postID: NSNumber?
+            var parentID: NSNumber?
+        }
```

Comment:
Always like a container type for information like this one. 

Do the properties need to be mutable?

Reply:
Maybe not? I used `var` out of habit without any extra thoughts. However, it's used as a constant though: `let pageData: PageData`.